libs/@guardian/atoms-rendering/src/YoutubeAtomPlayer.tsx

```diff
@@ -19,7 +19,7 @@ import type { AdTargeting, VideoEventKey } from './types';
 import type { PlayerListenerName } from './YouTubePlayer';
 import { YouTubePlayer } from './YouTubePlayer';
 
-declare class ImaManager {
+export declare class ImaManager {
```

Comment:
i think this should not use `declare`, to stop it being ambient itself

```suggestion
export class ImaManager {
```

Reply:
as in leave it as it was?

Reply:
ah ok great!